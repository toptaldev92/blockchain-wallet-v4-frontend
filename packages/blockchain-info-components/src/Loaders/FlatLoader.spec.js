import { shallow } from 'enzyme'
import React from 'react'
import toJson from 'enzyme-to-json'

import FlatLoader from './FlatLoader'

describe('FlatLoader component', () => {
  it('default renders correctly', () => {
    const component = shallow(
      <FlatLoader height='10px' width='10px' color='blue900' />
    )
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
