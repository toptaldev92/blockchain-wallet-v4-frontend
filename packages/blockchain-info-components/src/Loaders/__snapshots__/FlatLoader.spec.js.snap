// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlatLoader component default renders correctly 1`] = `
<styled.div
  height="10px"
  width="10px"
>
  <Styled(styled.div)
    color="brand-primary"
  />
  <Styled(styled.div)
    color="brand-primary"
  />
  <Styled(styled.div)
    color="brand-primary"
  />
  <Styled(styled.div)
    color="brand-primary"
  />
  <Styled(styled.div)
    color="brand-primary"
  />
</styled.div>
`;
